packages/rx_bloc_cli/mason_templates/bricks/lib_dev_menu/__brick__/lib/l10n/network_lookup/lib_dev_menu_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nLibDevMenuLookup extends I18nLibDevMenuLookup {
           bundledI18n = I18nLibDevMenu(I18nLibDevMenuLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nLibDevMenu(I18nLibDevMenuLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong